docs/labs/docker-plugin.rst

```diff
@@ -88,17 +88,17 @@ Upload these files to ``/etc/flocker/plugin.key`` and ``/etc/flocker/plugin.crt`
 
 Then perform the following instructions on each of the hosts where you want to install the Flocker Docker plugin.
 
-Install Experimental Docker
----------------------------
+Install Docker
+--------------
 
-Install the experimental build of Docker:
+Install Docker 1.8 or later:
```

Comment:
The following command installs the latest Docker, so this should say that. "Install the latest stable version of Docker". Or it should say something like, "Install Docker 1.8 or later, e.g. for the latest version of Docker:".